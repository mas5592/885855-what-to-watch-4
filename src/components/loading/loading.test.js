import React from 'react';
import renderer from 'react-test-renderer';
import Loading from './loading.jsx';

describe(`Loader`, () => {
  it(`Should render loading correctly`, () => {
    const tree = renderer
      .create(
          <Loading />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
