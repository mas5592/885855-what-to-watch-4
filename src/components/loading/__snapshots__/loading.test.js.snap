// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Should render loading correctly 1`] = `
<div
  className="user-page"
>
  <div
    className="sign-in user-page__content"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "center",
        "minHeight": "100vh",
        "width": "100%",
      }
    }
  >
    <div
      dangerouslySetInnerHTML={
        Object {
          "__html": undefined,
        }
      }
      style={
        Object {
          "fill": "#372824",
          "height": 64,
          "width": 64,
        }
      }
    />
  </div>
</div>
`;
